openssh/_mapdata/init.sls

```diff
@@ -5,14 +5,15 @@
 {%- set tplroot = tpldir.split('/')[0] %}
 {%- from tplroot ~ "/map.jinja" import mapdata with context %}
 
-{%- set output_file = '/tmp/salt_mapdata_dump.yaml' %}
+{%- set map = mapdata %}
+{%- do salt['log.debug']('### MAP.JINJA DUMP ###\n' ~ map | yaml(False)) %}
 
-{%- do salt['log.debug']( mapdata | yaml(False) ) %}
+{%- set output_file = '/tmp/salt_mapdata_dump.yaml' %}
```

Comment:
The restructuring of this section here is to find a common solution across formulas with and without the newest `map.jinja`.  That way, it will be easier to manage this file from the `ssf-formula`.

Reply:
For [template-formula](https://github.com/saltstack-formulas/template-formula/compare/master...baby-gnu:test/validate-map.jinja) I limit the difference between previous and new `map.jinja` to the `import` line:

```jinja
{%- from tplroot ~ "/map.jinja" import TEMPLATE as mapdata with context %}
```

Now, lets choose which one you prefer ;-)